hatch/app.py

```diff
@@ -45,13 +46,13 @@ def validate(request: Request, auth_token: str = Security(api_key_header)):
     try:
         token = AuthToken.verify(auth_token, config.JOB_SERVER_TOKEN, "hatch")
     except AuthToken.Expired as exc:
-        logger.info(str(exc))
+        logger.info(f"auth failed: {exc} ({auth_token})")
```

Comment:
We're not capturing the class of `exc` -- is that a problem?

Reply:
Possibly, I'll add it.